cilium/cmd/bgp_peer_get.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/api/v1/models"
+	"github.com/cilium/cilium/pkg/command"
+)
+
+var BgpPeersCmd = &cobra.Command{
+	Use:   "peers",
+	Short: "List current state of all peers",
+	Long:  "List state of all peers defined in CiliumBGPPeeringPolicy",
+	Run: func(cmd *cobra.Command, args []string) {
+		res, err := client.Bgp.GetBgpPeers(nil)
+		if err != nil {
+			Fatalf("cannot get peers list: %s\n", err)
+		}
+
+		if command.OutputOption() {
+			if err := command.PrintOutput(res.GetPayload()); err != nil {
+				Fatalf("error getting output in JSON: %s\n", err)
+			}
+		} else {
+			printSummary(res.GetPayload())
+		}
+	},
+}
+
+func printSummary(peers []*models.BgpPeer) {
+	// get new tab writer with predefined defaults
+	w := NewTabWriter()
+
+	// sort by local AS, if peers from same AS then sort by peer address.
+	sort.Slice(peers, func(i, j int) bool {
+		return peers[i].LocalAsn < peers[j].LocalAsn || peers[i].PeerAddress < peers[j].PeerAddress
+	})
+
+	fmt.Fprintln(w, "Local AS\tPeer AS\tPeer Address\tSession\tUptime\tFamily\tReceived\tAdvertised")
+	for _, peer := range peers {
+		fmt.Fprintf(w, "%d\t", peer.LocalAsn)
+		fmt.Fprintf(w, "%d\t", peer.PeerAsn)
+		fmt.Fprintf(w, "%s\t", peer.PeerAddress)
+		fmt.Fprintf(w, "%s\t", peer.SessionState)
+		fmt.Fprintf(w, "%s\t", time.Duration(peer.Uptime).String())
+
+		for i, afisafi := range peer.Families {
+			if i > 0 {
+				// move by 5 tabs to align with afi-safi
+				fmt.Fprintf(w, strings.Repeat("\t", 5))
+			}
+			// AFI and SAFI are concatenated for brevity
+			fmt.Fprintf(w, "%s_%s\t", afisafi.Afi, afisafi.Safi)
```

Comment:
Super Nit: I'd prefer ":" or "/" as an AFI/SAFI delimiter since some SAFI consists of multiple words, and we may want to use underscore as a word delimiter (e.g., MPLS_VPN https://pkg.go.dev/github.com/osrg/gobgp/v3@v3.13.0/api#Family_Safi).

Reply:
Updated